IntegrationTests/src/test/java/com/kenzie/appserver/controller/UserControllerTest.java

```diff
@@ -5,9 +5,9 @@
 import com.google.gson.Gson;
 import com.kenzie.appserver.IntegrationTest;
 import com.kenzie.appserver.controller.model.*;
```

Comment:
Avoid generic imports

Reply:
I've chosen to keep model.* as these packages are controlled by our team and are all usually used.